Include/XAsync.h

```diff
@@ -69,7 +69,7 @@ struct XAsyncBlock
 /// </summary>
 /// <param name='asyncBlock'>A pointer to the XAsyncBlock that was passed to the async call.</param>
 /// <param name='wait'>If true, XAsyncGetStatus waits until the async call either completes or is canceled.</param>
-STDAPI XAsyncGetStatus(
+HC_API XAsyncGetStatus(
```

Comment:
XAsync* is shared source from Brian, so changing it might not be wise

Reply:
Good point.  Another option would be to export the XAsync APIs (or even all the LHC APIs) with a .def file.  I find that to be slightly more annoying to maintain but maybe its worth it to leave the source untouched in this case.  Thoughts?

Reply:
def file is better.  this api list doesn't change often. i'd rather be able to keep the source in sync.